lib/amorail/client.rb

```diff
@@ -4,49 +4,50 @@
 require 'active_support'
 
 module Amorail
+  # Amorail http client
   class Client
-
     attr_accessor :cookies
 
     def initialize
       @host = Amorail.config.api_endpoint
-      @connect = Faraday.new(url: @host) do |faraday|           
-        faraday.adapter  Faraday.default_adapter
-        faraday.response :json, :content_type => /\bjson$/
+      @connect = Faraday.new(url: @host) do |faraday|
+        faraday.adapter Faraday.default_adapter
+        faraday.response :json, content_type: /\bjson$/
         faraday.use :instrumentation
       end
-
     end
 
     def connect
       @connect || self.class.new
     end
-    
+
     def authorize
-      response = post(Amorail.config.auth_url, {'USER_LOGIN' => Amorail.config.usermail, 'USER_HASH' => Amorail.config.api_key})
+      response = post(
+        Amorail.config.auth_url,
+        'USER_LOGIN' => Amorail.config.usermail,
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.